### PR TITLE
Convert from trytes to asciiString

### LINKED ISSUE #1301: Convert from trytes to ascii String
### Description
Add one method in Converter to translate trytes to ASCII text.

### Motivation
In Converter, there is a method called 'asciiToTrytes' to translate ASCII text to trytes, and the reverse method is missing.

### Requirements
translate trytes to ASCII text

### Open Questions (optional)
All the UTs for these conversions are missing, shall we add these UTs?

### Am I planning to do it myself with a PR?
Yes.

### PR DESCRIPTION
**Pull request by lhfbc**
# Description

Add one method in Converter to translate legal trytes to ASCII text. And change the location of 'allocateTritsForTrytes'

Fixes #1301 

## Type of change
- Enhancement (a non-breaking change which adds functionality)

# How Has This Been Tested?
- Add test case in ConverterTest

# Checklist:

- [ ] My code follows the style guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by lhfbc** [comment]
@kwek20  Would you help to review this PR?

File: src/test/java/com/iota/iri/utils/ConverterTest.java [hunk 1]
```diff
@@ -1,6 +1,18 @@
 package com.iota.iri.utils;
 
-public class ConverterTest {
+import org.junit.Assert;
+import org.junit.Test;
 
+public class ConverterTest {
+    @Test
+    public void testFromTextToTrites() {
```
Comment (GalRogozinski) [inline src/test/java/com/iota/iri/utils/ConverterTest.java]: Shouldn't it be `testTrytesToAscii`?

File: src/test/java/com/iota/iri/utils/ConverterTest.java [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+        String trytes = "LBCBKBSBCBKBKBRBCCAC9";
+        try {
+            String res = Converter.trytesToAscii(trytes);
+            Assert.assertEquals(res, "B9AI9AAHTR\0");
```
Comment (GalRogozinski) [inline src/test/java/com/iota/iri/utils/ConverterTest.java]: Maybe give a better text, to make the test easier to read. 
"EXPECTEDRESULT" will be a nice example :-)

File: src/test/java/com/iota/iri/utils/ConverterTest.java [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+        } catch(Exception e) {
+            e.printStackTrace();
+        }
```
Comment (GalRogozinski) [inline src/test/java/com/iota/iri/utils/ConverterTest.java]: So if an exception occurs you log it and don't fail the test?
You can just add `throws` to the method signature instead

**Comment by GalRogozinski** [comment]
Codacy is experiencing issues...
PR looks good to me

**Pull request merged**